<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>menu</class>
 <widget class="QMainWindow" name="menu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>576</width>
    <height>302</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>menu</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="importModel">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>40</y>
      <width>241</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string>导入模型</string>
    </property>
   </widget>
   <widget class="QPushButton" name="classicalModel">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>140</y>
      <width>241</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string>经典框架</string>
    </property>
   </widget>
   <widget class="QPushButton" name="designModel">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>40</y>
      <width>241</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string>自行设计</string>
    </property>
   </widget>
   <widget class="QPushButton" name="toolboxModel">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>140</y>
      <width>241</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string>数据集工具盒</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>576</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>菜单</string>
    </property>
   </widget>
   <addaction name="menu_2"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>designModel</sender>
   <signal>clicked()</signal>
   <receiver>menu</receiver>
   <slot>openDesign()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>210</x>
     <y>303</y>
    </hint>
    <hint type="destinationlabel">
     <x>291</x>
     <y>423</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>openDesign()</slot>
 </slots>
</ui>
